sdk/keyvault/Azure.Security.KeyVault.Certificates/ChangeLog.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 4.0.0-preview.4
+## 4.0.0-preview.4 (2019-10-07)
```

Comment:
preview.4 was already released.  This will be preview.5

Reply:
Thanks. Didn't realize that since we were tracking issues against the epic for preview 4.